src/_includes/docs/install/reqs/macos/zsh-config.md

```diff
@@ -0,0 +1,51 @@
+{% assign target = include.target %}
+
+<details markdown="1">
+<summary><strong>To verify your shell configuration, expand this section</strong></summary>
+
+Like most UNIX-like operating system, macOS can support multiple shells,
```

Comment:
This is the type of doc that tends to get out of date, and I don't think we need this page. We could probably just link to Apple's [Change the default shell in Terminal on Mac](https://support.apple.com/zh-sg/guide/terminal/trml113/mac) instead.

Reply:
This section wouldn't get out of date unless Apple changes the default shell (again). It's also hidden unless the reader needs to see it. That's because, well, they shouldn't unless they're running macOS 10.14 or earlier.

Reply:
In general this is not a doc that we need to maintain. In some cases it might be better to have no doc page at all, especially in this case anything we put here will already be covered by the official Apple / Zsh documentation. Need to be careful we don't increase our maintenance costs unnecessarily.

Reply:
I understand your point. I am willing to make any updates should they come. I stand by the change.